### PR TITLE
Improvements/more documentation

### LINKED ISSUE #37: Improvements on documentation
[ ] - Create a documentation to setup the price updater
[ ] - Improvements to README
 - What's commands are required
 - Add `$` in a text command to users know that command is runnable by binary

### PR DESCRIPTION
**Pull request by marioidival**
closes #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ $ go build -o priceupdater # or other name that you want
 
 ## Usage 
 
-The system has been designed to run through the commands listed above
+The system is designed to execute the commands listed above. The commands are listed to be executed in the order below.
```
Comment (marioidival) [inline README.md:22]:
```suggestion
The system has been designed to run through the commands listed above. The commands are listed to be executed in the order below.
```

File: README.md [hunk 2]
```diff
@@ -58,17 +58,18 @@ You can set up the configuration of the system by env file
 
 There are two commands related to providers:
 
-1. `change-provider --provider <provider-name>`
+1. [REQUIRED] `$ priceupdater change-prirotiry --provider <provider-name-1>,<provider-name-2>,<provider-name-3>`
```
Comment (marioidival) [inline README.md:61]:
```suggestion
1. [REQUIRED] `$ priceupdater change-prirotiry --priority <provider-name-1>,<provider-name-2>,<provider-name-3>`
```

File: SETUP_PRICE_UPDATER.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Before execute this steps, you should [configure](https://github.com/hermeznetwork/price-updater-service#configuration) the service.
+
+### Quick Guide
+
+```cmd=
+# First, define the priority execution of providers
+$ priceupdater change-priority --priority bifinex,uniswap
+
+# Second, load the configuration to providers
+$ priceupdater update-config --provider bitfinex --configFile pathToConfig.json
```
Comment (marioidival) [inline SETUP_PRICE_UPDATER.md:23]:
```suggestion
$ priceupdater update-config --provider bitfinex --configFile pathToConfig.json
$ priceupdater update-config --provider uniswap --configFile pathToConfig.json
```

**Review by invocamanman — APPROVED** [review]
nice thank you ^^

File: etc/priceupdater.service [hunk 4]
```diff
@@ -6,7 +6,7 @@ Type=simple
 Restart=always
 RestartSec=5
 KillMode=process
-ExecStart=price-updater server
+ExecStart=/home/ubuntu/price-updater-service/priceupdater server
```
Comment (ARR552) [inline etc/priceupdater.service:9]: Maybe, use a relative path is better

Comment (marioidival) [reply etc/priceupdater.service:9]: We had to use full path to facilitate the deployment on our servers

**Review by ARR552 — APPROVED** [review]

**Pull request merged**